app/assets/stylesheets/content/_table.sass

```diff
@@ -173,7 +173,7 @@ table.generic-table
       // become even wider.
       &.-max
```

Comment:
Please note that the comment is outdated as there is no interactive table any more so the column will not grow beyond `max-width`. This used to be true as the width style was set explicitly by js.